build-tools/integ/setup.integ.js

```diff
@@ -3,7 +3,7 @@
 const { configure } = require('@cloudscape-design/browser-test-tools/use-browser');
 
 configure({
-  browserName: 'ChromeHeadlessIntegration',
```

Comment:
@connorlanigan Were these changes intended to be committed?

Reply:
They were intended to be committed for easier debugging while you write the tests (because you can actually see what the browser does), but they should _not_ be part of the commit that goes into the main branch.